serve_mux.go

```diff
@@ -36,7 +35,7 @@ func (mux *ServeMux) match(q string, t uint16) Handler {
 		return nil
 	}
 
-	q = strings.ToLower(q)
+	q = CanonicalName(q)
```

Comment:
Could you also change `Fqdn(pattern)` to `CanonicalName(pattern)` in `Handle` and `HandleRemove`.

Reply:
Updated. :+1: